content/concepts/scalers/content.md

```diff
@@ -0,0 +1,37 @@
++++
+fragment = "content"
+weight = 100
+
+title = "Scalers"
+
+[sidebar]
+  sticky = true
++++
+
+## Overview
+
+Each event source has unique characteristics on how KEDA needs to connect to it and the types of metrics available to drive scaling.  KEDA is made up of many *scalers*, or simple extensions to the core KEDA operator that allows it to connect to an event source.  The full list of supported scalers can [be found here](/#scalers).
+
+Scalers are shipped with KEDA in two ways.  They can be packaged up and merged into the core KEDA operator, or can they can external processes or pods that KEDA can communicate with to drive scale.  Scalers that are packaged as part of the KEDA operator must be written as Go modules.  External scalers can be written in any language.
+
+## Contributing
+
+One of the easiest ways many people begin contributing to KEDA is by adding new event sources through scalers.  Below is guidance and recommendations when creating a scaler.
+
+### Building core or external scalers
+
+When building a scaler you'll need to determine if the scaler should be a core scaler shipped with every KEDA operator, or an external scaler that can be deployed and discovered independent of the core operator.  In general if you are building a scaler that connects to a popular event source, you should write them as core scalers in the `kedacore/keda` repo.  However, an external scaler may be a better fit if your scaler has one or many of the below characteristics.
+
+* Written in a language other than Go
+* Connects to an event source that is specific to a small set of users
+* Needs to be versioned or governed separately from the core operator
+
+## Support and Maintenance
+
+The core KEDA team and maintainers are responsible primarily for the KEDA operator.  Each scaler has an identified maintainer or sponsor responsible for scaler upkeep.  By default scalers are maintained with the designation **community**.  This means no direct organization or individual is accountable for bug fixes and upkeep.  Issues can and should still be filed, but the community will make a best effort to apply fixes.  If it's deemed by the community and maintainers that a scaler is no longer functioning properly, it may be removed from future releases.
+
+Any scaler can be submitted for sponsorship and maintainers by an organization or individual.  Maintaining a scaler means that as bugs and issues are filed, the maintainer of the scaler will be notified and it's expected issues will be triaged and acted on as appropriate by the maintainer.  If a maintainer becomes unresponsive or unreachable for an extended period of time, a scaler may be moved back to the **community** designation.
+
```

Comment:
What is the "extended period of time," ? :) Would it be worth adding a note that KEDA is a community project and everyone is encouraged to contribute a bug fix?

Also the notion of "maintainers" is quite strange for me, why the community has to wait for someone instead of fixing a bug? What in case of security issues?

Reply:
I don't think that it necessarily means that a bug fix can't be proposed by another user from the community, but that maintaner should handle it. If maintainer is not responsive then the scaler is moved back to the community (and could still accept fixes etc).

Reply:
> I don't think that it necessarily means that a bug fix can't be proposed by another user from the community

True, my thinking was that saying explicitly "feel free to fix it if you know how" encourage people to engage in the project.

Reply:
I think maintainer is an overloaded/confusing work here, I would call it "scaler owner" or "scaler maintainer"

I get both sides so would explicitly mention that it's if the scaler is not getting fixed by either community and/or scaler maintainer; only then we move it out as an add-on scaler or fully gone